### DISCUSSION TITLE
Error 504 on every page

### DISCUSSION THREAD

**Discussion by VirtualiteNL** [KelvinTegelaar/CIPP#2890] [Q&A]
I receive the error below on each page or button after completing CIPP SAM Setup Wizard.

<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Transitional//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'> <html xmlns='http://www.w3.org/1999/xhtml'> <head> <meta content='text/html; charset=utf-8' http-equiv='content-t...
504 - Request failed with status code 504

**Comment by KelvinTegelaar** [comment]
Support issue, support is only available for sponsors via the helpdesk email.
